### DISCUSSION TITLE
Part Time Remote Game Developer

### DISCUSSION THREAD

**Discussion by gorenbox** [pixijs/pixijs#7171] [Jobs]
Looking for a consultant / freelancer with extensive experience in Pixijs and game development to develop some mini-games for our company, IMBlox.
IMBlox a startup in the domain of marketing and bots and uses minigames to create fun experiences for our audience.  We are located in Tel-Aviv and Ukraine. Example mini-games that we could develop is a small packman game, a maze game, a wheel of fortune...The player plays for a min or 2 of fun and continues in the session whatever that may be. 
The first task would be to perfect our current games, and then to move on to create more games.
For more details, please feel free to contact me at dang@imblox.com.
Remote, flexible workhours and hours per week, competitive pay.

**Comment by bigtimebuddy** [comment]
@gorenbox Could you please provide more details, like company, examples of projects, how people should contact you?
